packages/office-ui-fabric-react/src/components/Slider/Slider.test.tsx

```diff
@@ -101,4 +106,34 @@ describe('Slider', () => {
 
     expect(component.find('label.ms-Label.ms-Slider-value').text()).toEqual(valueFormat(value));
   });
+
+  it('calls onChanged after keyboard event', () => {
+    jest.useFakeTimers();
+    const onChanged = jest.fn();
+
+    const container = document.createElement('div');
+    document.body.appendChild(container);
+
+    ReactDOM.render(<Slider label="slider" defaultValue={12} min={0} max={100} onChanged={onChanged} />, container);
+    const sliderSlideBox = container.querySelector('.ms-Slider-slideBox') as HTMLElement;
+
+    ReactTestUtils.Simulate.keyDown(sliderSlideBox, { which: KeyCodes.down });
+    ReactTestUtils.Simulate.keyDown(sliderSlideBox, { which: KeyCodes.down });
+    ReactTestUtils.Simulate.keyDown(sliderSlideBox, { which: KeyCodes.down });
+    ReactTestUtils.Simulate.keyDown(sliderSlideBox, { which: KeyCodes.up });
+    ReactTestUtils.Simulate.keyDown(sliderSlideBox, { which: KeyCodes.down });
+
+    expect(sliderSlideBox.getAttribute('aria-valuenow')).toEqual('9');
```

Comment:
Just a clarification question - why is aria-valuenow expected to be 9?

Reply:
The Slider value started at 12, so after 4 downs and 1 up, it should be at 9!

Reply:
And `aria-valuenow` is set to `value` in Slider.base